.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - alpha
+      - beta
+      - next
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: current
+      - name: Install pnpm and dependencies
+        uses: pnpm/action-setup@v2
+        with:
+          version: 7
+          run_install: true
+      - name: Build
+        run: pnpm build
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
```

Comment:
I see you added logic for publishing to eik in package.json, but you haven't added the eik step here - are you planning to do it in a separate PR?